virtualmicroscope/urls.py

```diff
@@ -24,9 +24,5 @@
 #
 
 from django.conf.urls import patterns
-from virtualmicroscope import views
```

Comment:
I meant not used directly in this file which is why it was flagged by `flake8`. Will look into that.